### DISCUSSION TITLE
Questions Regarding File Access and At a Glance

### DISCUSSION THREAD

**Discussion by StefiStarlite** [LawnchairLauncher/lawnchair#4648] [Q&A] [answered]
Everything I've tested has been working fairly well. I had two questions though:

1) when going into the icon packs and changing the app grid quantity for the Home pages, Dock, and App Drawer the file access notification appears. Not allowing access doesn't seem to restrict the particular function, what does Lawnchair use the file access permission for beyond maybe accessing icon files from the general files?

2) Regarding At a Glance, is it at all possible to implement calendar access from calendar apps besides Google's? Or if not, is there a way to redirect tapping on the widget to bring up a different calendar app instead (without having calendar entries from said app appear within the widget)?

I appreciate the help and thank you for all the work you all have put into the project!

**Comment by SuperDragonXD** [comment, marked as answer]
Hello! I know that this reply is a bit late, but regarding your questions:

1. This issue should now be resolved in Lawnchair 15 (there is now a button that allows you to grant access).
2. As of now, this isn't possible. Feel free to search our existing feature requests (or create a new one) though.
